### ISSUE TITLE
Inconsistent tunnel=yes rendering for streams

### ISSUE THREAD

**Issue by xkomczax** [gravitystorm/openstreetmap-carto#3676]
There is inconsistency for rendering tag+key values tunnel=yes on waterway=stream.

On zoom level 13, [the stream is not rendered](https://www.openstreetmap.org/#map=13/49.3835/16.7418).
On zoom level 14, [the stream is rendered as regular waterway](https://www.openstreetmap.org/#map=14/49.3840/16.7393).
On zoom level 15, [the stream is rendered with dashed line symbolizing tunnel](https://www.openstreetmap.org/#map=15/49.3838/16.7378).

My concern here is about zoom level 14: it should either not be rendered (the same as zoom 0-13) or rendered by a dashed line (the same as zoom level 15-19). Now it appears as "normal" waterway just for one zoom level, which is strange.

(just to note: the whole area I am linking is a karst zone with underground creeks/rivers, so it is tagged correctly).

**Comment by kocio-pl** [comment]
Thanks for the report. Are you interested in fixing this issue?

**Comment by jeisenbe** [comment]
It should be easy to fix this by removing rendering for streams underground / in tunnels at z14

**Comment by jeisenbe** [comment]
> the whole area I am linking is a karst zone with underground creeks/rivers, so it is tagged correctly

@imagico disagreed with the statement above. A `tunnel=yes` is usually a man-made feature. Natural waterways in caves are not normally mapped, because it is not possible for mappers to determine the course of the waterway underground with GPS.

Example from above:
https://www.openstreetmap.org/#map=14/49.3840/16.7393
![z15-suchdol-underground-streams](https://user-images.githubusercontent.com/42757252/68186967-c25ed080-ffe8-11e9-918c-8e97e3717089.png)

It's hard to see how the courses of these streams could have been mapped correctly.

**Comment by jragusa** [comment]
I fully agree, a karst system is not as simple as we can expect, you can also have diffuse infiltration and some part are inaccessible for humans, limiting accurate mapping. Note also that most of karst map are vertical (regarding to the depth) and not horizontal.

**Comment by jeisenbe** [comment]
@imagico your comment in https://github.com/gravitystorm/openstreetmap-carto/pull/3938#issuecomment-544477109 suggests that you might want to close this issue as not fixable. Is that correct?

**Comment by imagico** [comment]
The reason for this issue is that stream rendering has been removed from z13 without consideration for the zoom level progression otherwise leading to the observations of this issue.  Generally speaking i think rendering stream tunnels like normal streams at the lower zoom levels is a valid cartographic choice while not rendering them while normal streams are rendered otherwise leaving gaps (#3938) is not.

I would consider this issue to be about reviewing the zoom level progression of small waterways rendering for consistent and intuitive results after #3467.

**Comment by jeisenbe** [comment]
It would be good to open up a specific issue about the initial zoom level

At the time of #3467 I had hoped that we might just remove
intermittent=yes streams from z13 (which cause most of the visual
problems in the USA, since the imports are quite agressive about
labeling any sort of tiny intermittent watercourse) while keeping
permanent streams.

**Comment by imagico** [comment]
Well - the problems of current stream rendering were not solved by removing them from z13, they were just hidden.  There are other maps which render streams from z12 already with reasonable results so this is not really a matter of what is and is not possible to depict but of how to show it in a way that is well readable.

Starting intermittent streams one zoom level later is a possibility but it will only work if it is still an intuitive and well readable rendering otherwise.
